### DISCUSSION TITLE
Probleme de notification sur l'appli mobile

### DISCUSSION THREAD

**Discussion by Futeeee** [status-im/status-legacy#17833] [General]
Après avoir tester en demandant à plusieurs amis à moi qui ont un android et un iPhone de voir si le problème venait pas d'apple vu que je suis sur apple mais non c'est la même chose pour android. En gros sur mobile quand on reçoit une notif de message la notif est silencieuse c'est très bizarre on reçoit la notif que l'écran soit allumer ou non mais cette notifs ne fait pas de bruit (le téléphone est en mode sonnerie) je ne comprends pas pouquoi ca fait ca c'est la première fois que je vois ce probleme. Aussi il arrive qu'on reçoive des  notification de message alors qu'on a pas recu de message ducoup c'est surement un bug et ce problème est aussi arriver à un pote à moi qui utilise l'application. un dernier petit truc à rajouter j'utilise depuis plusieur année maintenant des aplli de message crypté et sur toute il y'a la fonctionnalité de l'autodéstruction des messages après 1h, 2h, 24h,.. etc si cette fonctionnalité peut être pressente avec le fait de pouvoir mettre un délai sur les poste qu'on met sur le "status" de l'appli status mdr l'appli serait fantastique. Je vous laisse ce message parce que j'aime beaucoup votre projet c'est pour ça que je prends de mon temps pour vous dire ce qu'il manquerait comme le fait d'appler sinom continuer comme ça, merci de m'avoir écouter bonne soirée à vous.

**Comment by ibrkhalil** [comment]
Basically, we need to investigate why PNs are not audible.
